pallets/thea-executor/src/weights.rs

```diff
@@ -0,0 +1,139 @@
+
+//! Autogenerated weights for `thea_executor`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-06-15, STEPS: `10`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `Krishnas-MacBook-Pro.local`, CPU: `<UNKNOWN>`
```

Comment:
I guess this should not be generated on the dev laptop and rather on dedicated server which we have for this...

Reply:
I'll generate everything on dedicated machine when all merged into Develop.
Pay attention to `weights.rs` files in Develop -> Mainnet branch PRs :)